src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/8.diff.sql

```diff
@@ -0,0 +1,432 @@
+/*
+We are making the following changes in this version of the schema
+-- Adding two bit columns to StringSearchParam table - IsMin and IsMax
+-- Updating the corresponding table value type to include these two new values
+-- Updating the Upsert stored procedure to handle these two new values
+*/
+
+CREATE TYPE dbo.StringSearchParamTableType_2 AS TABLE
+(
+    SearchParamId smallint NOT NULL,
+    Text nvarchar(256) COLLATE Latin1_General_100_CI_AI_SC NOT NULL,
+    TextOverflow nvarchar(max) COLLATE Latin1_General_100_CI_AI_SC NULL,
+    IsMin bit NOT NULL,
+    IsMax bit NOT NULL
+);
+
+ALTER TABLE dbo.StringSearchParam
+ADD IsMin bit NOT NULL,
+    IsMax bit NOT NULL,
+    CONSTRAINT IsMin_Constraint DEFAULT 0 FOR IsMin,
+    CONSTRAINT IsMax_Constraint DEFAULT 0 FOR IsMax;
+
+CREATE NONCLUSTERED INDEX IX_StringSearchParam_SearchParamId_Text
+ON dbo.StringSearchParam
+(
+    SearchParamId,
+    Text
+)
+INCLUDE
+(
+    ResourceTypeId,
+    TextOverflow, -- workaround for https://support.microsoft.com/en-gb/help/3051225/a-filtered-index-that-you-create-together-with-the-is-null-predicate-i
+    IsMin,
+    IsMax
+)
+WHERE IsHistory = 0 AND TextOverflow IS NULL
+WITH 
+(
+    DATA_COMPRESSION = PAGE,
+    DROP_EXISTING = ON
+)
+
+CREATE NONCLUSTERED INDEX IX_StringSearchParam_SearchParamId_TextWithOverflow
+ON dbo.StringSearchParam
+(
+    SearchParamId,
+    Text
+)
+INCLUDE
+(
+    ResourceTypeId,
+    IsMin,
+    IsMax
+)
+WHERE IsHistory = 0 AND TextOverflow IS NOT NULL
+WITH 
+(
+    DATA_COMPRESSION = PAGE,
+    DROP_EXISTING = ON
```

Comment:
Please add `ONLINE = ON` so that the table is not exclusively blocked.

Reply:
Updated both `CREATE INDEX` commands with  addition of `ONLINE = ON`